lib/vue-scroll.js

```diff
@@ -60,15 +66,39 @@
                   
                 if(typeof Q._initialized == 'undefined'){
                     
-                  Q.prototype.bind = (function(element, eventType, func){
-                    var funcs;
-                    
-                    if(elements.indexOf(element) < 0){
+                  Q.prototype.bind = (function(element, eventType, func, arg){
+                    var funcs, i, length = elements.length, throttle;
+
+                    if(arg){
+                        var lastCallTimestamp, timeout = null, value, pos, caller = function(){
```

Comment:
@wangpin34 I agree with you, but when you use <b>indexOf</b> with objects you need to know exact values of each field. I made this to save both throttling and original functions in array. If you still want to use <b>indexOf</b> you will need two arrays to store both throttle\debounce and original functions.  And you'll need to add polyfill for it.